src/plugins/downloader/templates/download-manager.css

```
@@ -0,0 +1,629 @@
+  width: 380px;
+  max-height: 500px;
+  z-index: 100000;
+  display: flex;
+  flex-direction: column;
+  background: rgba(22, 22, 28, 0.98);
+  backdrop-filter: blur(20px) saturate(1.5);
+  -webkit-backdrop-filter: blur(20px) saturate(1.5);
+  border: 1px solid rgba(255, 255, 255, 0.08);
+  border-radius: 12px;
+  box-shadow: 0 16px 48px rgba(0, 0, 0, 0.5), 0 0 0 1px rgba(0, 0, 0, 0.2);
+  font-family: 'Google Sans', 'Roboto', sans-serif;
```

Stylelint (17.7.0)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)